invenio_records_resources/resource_unit.py

```diff
@@ -6,12 +6,11 @@
 # modify it under the terms of the MIT License; see LICENSE file for more
 # details.
 
-"""State API."""
+"""Resource unit."""
```

Comment:
minor: since there are more than one in the file `resource_units.py`? (final *s*) It's true we have no rule on this (or at leas we don't follow). But files with more than one are e.g. `responses.py` files with one `service.py`. OCD kicked in sorry :)

Reply:
:100: You are absolutely right. I did it for `service/` -> `services/` but forgot this one. On it!